spec.html

```diff
@@ -23028,8 +23116,129 @@ <h1>InnerModuleEvaluation ( _module_, _stack_, _index_ )</h1>
                   1. Assert: _requiredModule_ is a Cyclic Module Record.
                   1. Set _requiredModule_.[[Status]] to ~evaluated~.
```

Comment:
This is missing an if-else that sets the [[Status]] to `~evaluating-async~` if [[Async]] is true.

Reply:
Good catch, this is already in https://github.com/tc39/proposal-top-level-await/blob/main/spec.html#L607.

Reply:
sorry, missed this while making the merge